lib/DismissiblePrompt/.specs/DismissiblePrompt.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, it, vi } from "vitest";
+import React from "react";
+import DismissiblePrompt from "../";
+import Button from "../../Button";
+
+import { render, fireEvent, screen } from "@testing-library/react";
+
+describe("DismissiblePrompt", () => {
+  let wrapper: any;
+
+  const onDismissSpy = vi.fn();
+
+  beforeAll(() => {});
+
+  test("renders children", () => {
+    render(
+      <DismissiblePrompt
+        data-testid="test-dismissible-prompt"
+        onDismiss={onDismissSpy}
+        className={"hellooooo"}
+      >
+        Hello world
+      </DismissiblePrompt>
+    );
+    expect(screen.getByText("Hello world"));
+    expect(
+      screen
+        .getByTestId("test-dismissible-prompt")
+        .classList.contains("hellooooo")
+    ).toBeTruthy();
```

Comment:
I'm not 100% sure on this opinion, but I'll share it anyway in case it's helpful (doesn't necessarily need actioning)...

If asserting that the class has been added, should we use a screen query that grabs element by the given class and check they exist that way?

Reply:
I tried this at first but couldn't figure out how to target the DOM, eg `screen.getElementsByClassName` and `screen.querySelectorAll` you can't do.

Looking through docs but unsure what would be used to specifically target classnames @timbryandev

Reply:
I dont think we need to test that the className prop works, thats just core react so we can probs just ditch imo!